pkg/harvester/dialog/HarvesterQuotaDialog.vue

```diff
@@ -88,7 +90,7 @@ export default {
         v-int-number
         :label="t('harvester.snapshot.totalSnapshotSize')"
         :disabled="false"
-        :mode="create"
+        :mode="_EDIT"
```

Comment:
How about remove `:mode` using default  __EDIT in [UnitInput](https://github.com/harvester/dashboard/blob/9cf422b3a553731f173252f26953be09a2b38f97/shell/components/form/UnitInput.vue#L80) ?

Reply:
Even better, thanks.